systems/plants/TimeSteppingRigidBodyManipulator.m

```diff
@@ -620,7 +620,8 @@ function checkDirty(obj)
 
```

Comment:
Seems like there was some discussion on the specifics of the fastQP setup in #668.  Using that formulation, fastQP fails in only ~80/1001 calls in runAtlasBalancing.  

Using the proposed solution above, it's failing in 915/1001 calls.  I think the sliding motion is happening because we aren't specifying the complementarity constraints in the fastQP formulation.  The objective function is just minimizing the norm of z.

By setting

    M*z + w = 0

It's almost like specifying a pseudo-complementarity constraint since 

    z' * (M * z + w) = 0

Will be satisfied based on the value of ```M * z + w``` alone.  In the true LCP, z can cause the above equality to be true even for non-zero ```M * z + w``` .